gk/main.c

```diff
@@ -1771,6 +1791,16 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 				ret, fe->state);
 	}
 
+	ret = gk_solicitor_enqueue_bulk(gk_conf->sol_conf, req_bufs, req_prio,
+		num_reqs);
+	if (ret <= 0) {
```

Comment:
Shouldn't we test that `num_reqs` is greater than zero before calling `gk_solicitor_enqueue_bulk()`?